API_Nasa_Call.py

```diff
@@ -1 +1,89 @@
-# Calls Nasa's API to return a picture of the APOD and return the link to the image for now until Flask is setup.
\ No newline at end of file
+# Calls Nasa's API to return a picture of the APOD and return the link to the image for now until Flask is setup.
+
+'''This API ask a user to enter a specific date between beginning_date which is 1995-6-16 until today_date the current day
+ and fetch a picture from APOD url and show the picture in web browser from now until we decide to use Flask '''
+
+import requests
+import datetime
+from datetime import date
+import webbrowser 
+
+'''This function set a range of valid dates 
+    if the user enter the right dates it will return the date_pic if not it will ask them to enter the right dates'''
+    
+    # Get a picture for a specific date
+def get_date(): 
+
+    today_date = date.today() 
+    beginning_date = datetime.date(1995, 6, 16)  
+    
+    # recursively iterate over a list of False.
```

Comment:
What does this comment mean?

Reply:
The loop will only stop when the condition is false I might explained it wrong.